README.md

```diff
@@ -401,16 +407,23 @@ Total: 100 / 100
 ```
 
 Extending the previous example, we can manually set the time of submission through `proxy-time`.
-We can use a [Unix Epoch time converter](https://www.epochconverter.com/) to change the target time
-of March, 4th 2025 at 12:00 PM Pacific Time to milliseconds from the Unix Epoch, `1741118400000`:
+We will use the time `1741118400000` (March 4th, 2025 at 12:00 PM Pacific Time).
 ```sh
 python3 -m autograder.cli.courses.assignments.submissions.proxy.submit --proxy-email student@test.edulinq.org --proxy-time 1741118400000 my_file.py
```

Comment:
Move `--proxy-time` to be the first option so it is more likely to show when width is limited.

(Same with the other second example, move the new param up.)